frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineVisualizationTask.tsx

```diff
@@ -124,7 +130,7 @@ const TaskComponent: React.FC<TaskProps> = ({
           'is-text-center': !isPipelineRun,
         })}
       >
-        <div className="odc-pipeline-vis-task__title">{visualName}</div>
+        <div className="odc-pipeline-vis-task__title">{truncatedName}</div>
```

Comment:
Truncation as requested by UX
![image](https://user-images.githubusercontent.com/8126518/75258463-7bbdfd80-57b4-11ea-9cdd-059b80594836.png)

Reply:
When there are wider characters, it fails to properly truncate for a fixed size. Will remove this change.

![image](https://user-images.githubusercontent.com/8126518/75293407-1853c000-57f4-11ea-8770-bc88122e1086.png)

Reply:
Removed.